k9mail-library/src/main/java/com/fsck/k9/mail/power/TracingPowerManager.java

```diff
@@ -25,7 +23,7 @@ public static synchronized TracingPowerManager getPowerManager(Context context)
         Context appContext = context.getApplicationContext();
         if (tracingPowerManager == null) {
             if (K9MailLib.isDebug()) {
```

Comment:
Hmm. Those conditionals relate to properties in K-9. Should we remove those too? @cketti ?